glue/core/data_exporters/gridded_fits.py

```diff
@@ -28,6 +44,8 @@ def fits_writer(filename, data, components=None):
     else:
         mask = None
 
+    data_header = data.coords.header if hasattr(data.coords, "header") else None
```

Comment:
A more robust way here would be to check if ``data.coords`` is an instance of ``glue.core.coordinates. WCSCoordinates`` (rather than just checking whether ``header`` exists).